CONTRIBUTING.md

```diff
@@ -40,6 +40,14 @@ CONTRIBUTING
 
 ## Getting Started
 
+Make sure you have at least [Node.js v6][11]:
+
+```sh
+node -v
+
+v6.2.1
```

Comment:
Added a note to contributing to use Node 6 since babel-node is no longer transpiling build and config code.